blacklight-cornell/app/helpers/browse_helper.rb

```diff
@@ -1,28 +1,33 @@
 module BrowseHelper
 
-def search_field(headingType)
-	if headingType == "Personal Name" 
-		search_field = 'pers'
-	elsif headingType=="Corporate Name"
-		search_field = 'corp'
-	elsif headingType == "Event"
-		search_field = 'event'
-	elsif headingType == 'Geographic Name'
-		search_field = "geo"
-	elsif headingType == 'Chronological Term'
-		search_field = 'era'
-	elsif headingType == 'Genre/Form Term'
-		search_field = 'genr'
-	elsif headingType == "Topical Term"
-		search_field = 'topic'
-	elsif headingType=='Work'
-		search_field='work'
-	else search_field='all_fields'
+  def search_field(heading_type)
+    case heading_type
+    when 'Personal Name'
+      'pers'
+    when 'Corporate Name'
+      'corp'
+    when 'Event'
+      'event'
+    when 'Geographic Name'
+      'geo'
+    when 'Chronological Term'
+      'era'
+    when 'Genre/Form Term'
+      'genr'
+    when 'Topical Term'
+      'topic'
+    when 'Work'
+      'work'
+    else
+      'all_fields'
+    end
+  end
 
-	end
+  def browse_search_field(browse_type, heading_type)
```

Comment:
I shouldn't have merged this into dev. It's not going to break anything, but it's not doing anything and nothing's calling it. I'll just create a branch for removing it and can merge it in in the next sprint, so it doesn't delay the deploy.